Cartfile.private

```diff
@@ -1 +1 @@
-github "AliSoftware/OHHTTPStubs" "swift-3.0"
+github "AliSoftware/OHHTTPStubs" "master"
```

Comment:
We aren’t pinning to master. See Cartfile.resolved, which specifies the exact commit hash that `carthage bootstrap` will always fetch. This just specifies our intent so that we can easily `carthage update` to grab the latest master in the future. I’m checking this change in now because AliSoftware/OHHTTPStubs#240 has landed and the Swift 3 branch will probably go away soon, stranding this repository.